content/en/development/design-docs/clb-implementation.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "StandardV2 Load Balancer - Container Based Backendpool"
+linkTitle: "StandardV2 Load Balancer - Container Based Backendpool"
+type: docs
+description: >
+    Implement new Load Balancer sku - StandardV2
```

Comment:
This is great! Could you add a little more details on the doc, including benefits (compared to SLB), how it works (e.g. add a simple diagram to show its datapath), how to use (e.g. config and service spec sample)?